.github/workflows/os-zoo.yml

```diff
@@ -8,6 +8,7 @@
 name: OS Zoo CI
 
 on:
+  workflow_dispatch:
```

Comment:
BTW: how is this supposed to work?

Reply:
workflow_dispatch is the manual dispatch event trigger in github. By adding it, a dropdown will appear here:
https://github.com/openssl/openssl/actions
in the upper right corner, allowing us to select this workflow for execution manually.  I added it because normally this is only run overnight, so we miss alot of stuff in it.  It will let us run it by hand when we need to debug something in the future

Reply:
but wont it have the same problem that we had with running it on pull_request?

Reply:
oh, I see what you're saying, because we always pull the active branches, any changes we want to test won't get picked up.

I can fix that though, I can modify the dispatch event to accept an input that is the PR branch to test, and if set, just run on that branch rather than all the active branches.  I'll take care of that in a subsequent PR (I'd like to get this merged since its fixing a CI break)

Reply:
ok, FYI I can already re-run any OS-Zoo test run,  I just tried it and it works already.
even without this change.

Reply:
Please consider doing what I did here https://github.com/openssl/openssl/pull/24537/files - i.e. that job runs when `extended tests` label is set on a PR, otherwise it is skipped. The if condition would have to be slightly modified to not check the label when the job is run on schedule.

Maybe this is material for separate subsequent PR.